ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -51,7 +51,7 @@
         </v-row>
         <v-row>
           <v-spacer></v-spacer>
-          <v-col cols="9" class="pl-3">
+          <v-col cols="9" class="pl-1">
```

Comment:
Does the 8px adjustment still keep it in alignment with the Land Lease or Ownership Checkbox?

![Screen Shot 2023-03-21 at 5 11 09 PM](https://user-images.githubusercontent.com/53542131/226769498-540d34f3-c295-42f1-8bcd-59ab5602663c.png)

Reply:
What is this voodoo that keeps it in alignment, unless the land lease is moved too?

Reply:
The voodoo is no longer voodooing. Will confer with Sara for what to do with the land lease checkbox

Reply:
Moved Land Lease checkbox in line

Reply:
hahahah right on man, sounds good 👍